.vib/harbor-exporter/goss/harbor-exporter.yaml

```diff
@@ -0,0 +1,25 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /var/log/jobs:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+command:
+  # Ensure swagger binary doesn't exist
+  check-swagger-binary:
+    exec: ls /opt/bitnami/harbor-exporter/bin/swagger 2>/dev/null
+    exit-status: 2
+  # Ensure a set of directories exist and the non-root user has write privileges to them
+  check-directories-exist-with-user:
+    exec: ls -dl /etc/ssl/certs /etc/pki/tls/certs/ 2>/dev/null | grep "drwxrwxr-x.*harbor"
+    exit-status: 0
+  # Ensure permissions for Internal TLS
+  check-permissions-system-certs:
+    exec: ls -l /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.trust.crt /etc/ssl/certs/ca-certificates.crt 2>/dev/null | grep ".*-rw-rw-r--"
+    exit-status: 0
```

Comment:
Why use 'ls' and 'grep' instead of a goss `file` test?

Reply:
We don't know which of the files exists or not, so we cannot perform this check with the format you propose. So the use of `file` isn't possible in this case.